pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [metadata]
```

Comment:
not sure if the metadata are used any more, would be better to convert to `project` tag
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#basic-information